server/views/embed.py

```diff
@@ -30,7 +30,12 @@ def embed(request, wfmodule_id):
                 wf_module = None
 
     if wf_module:
-        ctx = JsonizeContext(request.user, request.session, request.locale_id)
+        ctx = JsonizeContext(
+            request.user,
+            request.session,
+            request.locale_id,
+            dict(MODULE_REGISTRY.all_latest()),
```

Comment:
No need for the registry. Just use `{module_zipfile.module_id: module_zipfile}`.